ci/lint.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.17.1
```

Comment:
This should be set to latest - we should probably update the template in dp-cli to match